scripts/convert_animatediff_motion_module_to_diffusers.py

```diff
@@ -48,4 +48,4 @@ def get_args():
     # skip loading position embeddings
     adapter.load_state_dict(conv_state_dict, strict=False)
     adapter.save_pretrained(args.output_path)
-    adapter.save_pretrained(args.output_path, variant="fp16", torch_dtype=torch.float16)
+    adapter.to(torch.float16).save_pretrained(args.output_path, variant="fp16")
```

Comment:
Shouldn't this be made configurable? Folks might not want to serialize the model in fp16.

Reply:
Updated